org.eclipse.scout.rt.api.data/src/main/java/org/eclipse/scout/rt/api/data/security/PermissionCollectionType.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2010, 2023 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.api.data.security;
+
+import org.eclipse.scout.rt.dataobject.enumeration.IEnum;
+import org.eclipse.scout.rt.platform.util.Assertions.AssertionException;
+
+public enum PermissionCollectionType implements IEnum {
+  DEFAULT("DEFAULT"),
+  ALL("ALL"),
+  NONE("NONE");
+
+  private final String m_stringValue;
+
+  PermissionCollectionType(String stringValue) {
+    m_stringValue = stringValue;
+  }
+
+  @Override
+  public String stringValue() {
+    return m_stringValue;
+  }
+
+  public static PermissionCollectionType resolve(String value) {
```

Comment:
resolve implementation may be omitted, as long as enum names and id-strings correspond
the possibility to add a resolve funtion is just in case of a later change in values which requires a on-the-fly migration by a custom resolve method implementation